<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhraseUniqueWordsForm</class>
 <widget class="QWidget" name="PhraseUniqueWordsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="WordListGeneralControlsForm" name="wordList_general_controls" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="NgramPUWRationlizeForm" name="ngram_puw_rationlize" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="GNGTableView" name="tableView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GNGTableView</class>
   <extends>QTableView</extends>
   <header location="global">gng_tableview.h</header>
  </customwidget>
  <customwidget>
   <class>WordListGeneralControlsForm</class>
   <extends>QWidget</extends>
   <header location="global">wordlistgeneralcontrolsform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NgramPUWRationlizeForm</class>
   <extends>QWidget</extends>
   <header location="global">ngrampuwrationlizeform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
